api/v1/controllers/auditEvents.js

```diff
@@ -14,10 +14,48 @@
 const featureToggles = require('feature-toggles');
 const helper = require('../helpers/nouns/auditEvents');
 const u = require('../helpers/verbs/utils');
-const httpStatus = require('../constants').httpStatus;
+const constants = require('../constants');
+const httpStatus = constants.httpStatus;
 const doFind = require('../helpers/verbs/doFind');
 const doGet = require('../helpers/verbs/doGet');
+const apiErrors = require('../apiErrors');
+
+/**
+ * Function to validate the query parameters in the request
+ * @param  {Object} params  - The request params
+ */
+function validateQueryParams(params) {
```

Comment:
are we validating that endAt >= startAt?

Reply:
Missed this validation. Will do it.